raw-data/UW/uw_inventories.tsv

```diff
@@ -0,0 +1,55 @@
+FileNames	Glottocode	LanguageName	Specific Dialect	Phonemes	Comments	Allophones	ExplanationOfAllophones	Notes
+roseslabrada2015mako.pdf		Mako		i		i		
+						ĩ	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	
+				e		e		
+						ẽ	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	
+				ɨ		ɨ		
+						ɨ̃	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	
+				a	low central vowel	a		
+						ã	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	
+				u		u		
+						ũ	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	
+				o		o		
+						õ	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	
+				<ə>	only appears in past tense suffix /-tə/			
+				ĩ				
+				ẽ				
+				ɨ̃				
+				ã	low central nasal vowel			
+				ũ				
+				õ				
```

Comment:
sorry I didn't notice this the first time; if a phoneme doesn't have any allophones we just put a copy of the phoneme itself in the allophone column (i.e., there should be no empty cells in the allophone column, except for the blank line between different inventories.

Reply:
@drammock -- We will need to extend the aggregation script for these inventories, as well as the ones being produced by JM. So perhaps better to assign IDs once a batch is finished, so we're not jumping back and forth between newly add sources for ID assignment.